### PR TITLE
update travis config

### PR DESCRIPTION
**Pull request by gurneyalex**
use UNIT_TEST rather than maintain complex include/exclude rules

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,13 +6,8 @@ python:
...
 env:
-  - VERSION="7.0" ODOO_REPO="odoo/odoo" EXCLUDE="account_constraints,async_move_line_importer,account_move_validation_improvement,account_default_draft_move,account_move_batch_validate"
-  - VERSION="7.0" ODOO_REPO="odoo/odoo" INCLUDE="account_constraints,account_move_validation_improvement,account_default_draft_move,account_move_batch_validate"
-  - VERSION="7.0" ODOO_REPO="odoo/odoo" INCLUDE="async_move_line_importer"
-
-  - VERSION="7.0" ODOO_REPO="OCA/OCB"   EXCLUDE="account_constraints, async_move_line_importer,account_move_validation_improvement,account_default_draft_move,account_move_batch_validate"
-  - VERSION="7.0" ODOO_REPO="OCA/OCB"   INCLUDE="account_constraints,account_move_validation_improvement,account_default_draft_move,account_move_batch_validate"
-  - VERSION="7.0" ODOO_REPO="OCA/OCB" INCLUDE="async_move_line_importer"
+  - VERSION="7.0" ODOO_REPO="odoo/odoo" UNIT_TEST=1
+  - VERSION="7.0" ODOO_REPO="odoo/odoo" UNIT_TEST=1
```
Comment (tafaRU) [inline .travis.yml]: I think you have to replace odoo/odoo by OCA/OCB ;-)

**Comment by tafaRU** [comment]
Now LGTM, so :+1:

**Comment by eLBati** [comment]
As reference, see @pedrobaeza's argument at https://github.com/OCA/purchase-workflow/pull/51#issuecomment-61775232

**Comment by bwrsandman** [comment]
While you're at it, please remove .coveragerc as it is supplied by MQT

**Comment by bwrsandman** [comment]
:+1:

**Comment by guewen** [comment]
:+1:

**Comment by gurneyalex** [comment]
Note: builds with this config take way to much time, and we are finding this difficult to cope with in OCA/vertical-ngo and we are reverting the change. Maybe you'll want to do the same here.

**Pull request merged**